src/importexport/guitarpro/internal/importgtp-gp5.cpp

```diff
@@ -639,7 +641,7 @@ void GuitarPro5::readMeasures(int /*startingTempo*/)
         }
     }
 
-    if (gpLyrics.segments.size()) {
+    if (!gpLyrics.segments.empty()) {
```

Comment:
That should be the same thing, shouldn't it?

Reply:
it is indeed, didn't change the logic, just looks more clear